crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -186,8 +177,10 @@ An alternative to allocating this on the heap is to provide a reference to a sta
 Usage restrictions
 ------------------
 
-On the nRF9160 SiP, the nrf_cc3xx_mbedcrypto library is restricted to only work in secure processing environment.
-The library uses mutexes to ensure single usage of hardware modules.
+The library can not be used in the non-secure domain of an application that uses ARM TrustZone.
```

Comment:
Change so that "nonsecure" is one word.

Reply:
Non-secure is a term.

I'm sure some people have written nonsecure, but non-secure is dominant.

https://www.google.com/search?q=trustzone+non-secure&sxsrf=AOaemvLSh3VK6eKxdzZf1v0tN11GT0Q1EA%3A1637339586627&ei=wtGXYdDkJb-FwPAPlOOfgAo&ved=0ahUKEwjQzb-j7aT0AhW_AhAIHZTxB6AQ4dUDCA4&uact=5&oq=trustzone+non-secure&gs_lcp=Cgdnd3Mtd2l6EAM6BwgAEEcQsANKBAhBGABQkR5YkR5g2CNoAXACeACAAWuIAWuSAQMwLjGYAQCgAQHIAQjAAQE&sclient=gws-wiz

Reply:
Yes, you are correct. Sorry about that!